df.military.xml

```diff
@@ -586,8 +586,8 @@
                 <int32_t name="nemesis_id" init-value='-1'/>
                 <int32_t name="hunger_timer"/>
                 <int32_t name="thirst_timer"/>
+                <int32_t name="sleepiness_timer"/>
```

Comment:
Interesting, this is a subset of `unit.counters2` (but not identical). In any case, the only tools that use `sleepiness_timer` appear to use the one in counters2, not this one, so this shouldn't break anything.